### DISCUSSION TITLE
Should we provide  NIST IR 8477 Comments ?

### DISCUSSION THREAD

**Discussion by smartopian** [KantaraInitiative/ancr-wg#14] [General]
Title: _Mapping Relationships Between Documentary Standards,Regulations, Frameworks, and Guidelines :Developing Cybersecurity and Privacy Concept Mappings_
[NIST.IR.8477.ipd.pdf](https://github.com/KantaraInitiative/WG-ANCR/files/12379090/NIST.IR.8477.ipd.pdf)

This seems like the instructions for how to map things wrt privacy, transparency and records of processing.    We need someone well versed in NIST security controls

**Comment by IDmachines** [comment]
So, I am very familiar with those controls effectively the 800-53 superset and then those in the Cybersecurity Framework and the Privacy Framework. These can map to 29100 and 27001/2.  See attachment for the mapping to 27001.

 

They are very data protection and identity management centric from the organization perspective identity provider and controller as you know.

 

I don’t want to take too much time with this but it would be useful, imo, to see the extent to which the TPIs and other aspects of the consensus protocol, controller conformance and compliance are accounted here.

From: Mark Lizar ***@***.***> 
Sent: Friday, August 18, 2023 8:18 AM
To: KantaraInitiative/WG-ANCR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [KantaraInitiative/WG-ANCR] Should we provide NIST IR 8477 Comments ? (Discussion #14)

 

Title: Mapping Relationships Between Documentary Standards,Regulations, Frameworks, and Guidelines :Developing Cybersecurity and Privacy Concept Mappings
NIST.IR.8477.ipd.pdf <https://github.com/KantaraInitiative/WG-ANCR/files/12379090/NIST.IR.8477.ipd.pdf> 

This seems like the instructions for how to map things wrt privacy, transparency and records of processing. We need someone well versed in NIST security controls

**Reply by smartopian** [reply]
This seems to bring up a couple o things, which likely warrant a dedicated thread in ANCR with regards to NIST 
1. we have an action to map  NIST 800-63- transparency and consent profile - so we can run some TPI's on it and present to Kantara
2. there are some  Notice and Transparency Gaps,  that we should document 
3. there is security gaps and cyber risks addressed by the ANCR Framework - which we should report
4. should there be   a receipt wallet with NIST mapping for DHS use case

**Reply by smartopian** [reply]
Also, now that 27560 is published
we can  map 29184 controls and structure to the ANCR Record Framework to be used in TPI's - and then see how NIST does against 29184 :-) that would be interesting

**Comment by IDmachines** [comment]
Very much so, I had searched the document, before your email, transparency is only mentioned twice and not in a relevant privacy context as we consider it. Consent and Notice do not appear in the document.

**Comment by IDmachines** [comment]
The Cybersecurity Framework 2.0 is open for comment, that is another open NIST venue for input..

 

https://csrc.nist.gov/pubs/cswp/29/the-nist-cybersecurity-framework-20/ipd
